multinet/api/views/workspace.py

```diff
@@ -39,18 +65,86 @@ def create(self, request):
         serializer = WorkspaceCreateSerializer(data=request.data)
         serializer.is_valid(raise_exception=True)
 
+        is_public = False
+        if 'public' in serializer.validated_data:
+            is_public = serializer.validated_data['public']
```

Comment:
If this is `Dict.get()`, then it takes a second parameter for the default value (i.e. `.get('public', False)`).

Is there a reason that `validated_data` doesn't *always* contain a `public` key?

Reply:
Since `public = False` by default, it is not required to be in the request's data for creating a new workspace. I'll go with the `Dict.get('public', False)` method here.